bin/kubedecode

```diff
@@ -20,7 +20,7 @@ fi
 SECRET="$1"
 
 if [[ -z "${2// }" ]]; then
-  NAMESPACE=$(kubectl config get-contexts --no-headers | tr -s ' ' | cut -d ' ' -f5)
+  NAMESPACE=$(kubectl config get-contexts  $(kubectl config current-context) --no-headers | tr -s ' ' | cut -d ' ' -f5)
```

Comment:
can't this just be `NAMESPACE=$(kubectl config current-context)`?

Reply:
No @Jengah . Because the `context` is just a friendly name that map to a `cluster` and a `namespace`.

We would like to get the **namespace** of the **current context**.

Reply:
Ah, right. I always add a namespace matching my context in my kube config and forget that that isn't automatic behavior.

Thanks!

Reply:
ping

Reply:
news?

Reply:
ping